cocotb/share/makefiles/Makefile.sim

```diff
@@ -100,4 +100,6 @@ ifeq ($(HAVE_SIMULATOR),0)
     $(error "Couldn't find makefile for simulator: "$(SIM_LOWERCASE)"! Available simulators: $(AVAILABLE_SIMULATORS)")
 endif
 
+export LIBPYTHON_LOC := $(shell find_libpython)
```

Comment:
Add error checking in case `find_libpython` does not find libpython. Could mention something along the lines of "libpython not found. You may not have the python development libraries installed, or find_libpython might not be able to find a libpython for your environment."